mappyfile/utils.py

```diff
@@ -632,10 +636,11 @@ def _save(output_file, string):
         f.write(string)
 
 
-def _pprint(d, indent, spacer, quote, newlinechar, end_comment, align_values, **kwargs):
+def _pprint(d, indent, spacer, quote, newlinechar, end_comment, align_values, separate_complex_types, **kwargs):
```

Comment:
`_pprint` is also called by the `dump` and `dumps` functions so the `separate_complex_types` parameter (and updated docstring) will need to be added to these also

Reply:
Hey,

this could take a bit longer. I'm developing with Python3 but tests for Python 2 fail because method "move_to_end" I used is only available since Python3.
Tried to add it but then it still fails for different reason.
If you have an idea - I have to dig deeper into that but this can take a bit.

Reply:
Looks like the `self.__map` doesn't exist. The answer at https://stackoverflow.com/a/54146075/179520 looks clean enough - using py3 `move_to_end` when available, and recreating the dict in Python2. For info I'll be dropping Python2 in a few months (the main dependencies of mappyfile have already done this).